connectors/dds4ccm/impl/DDS_Base_Connector_T.cpp

```diff
@@ -98,8 +98,9 @@ DDS_Base_Connector_T<CCM_TYPE>::create_dds_participant_factory (void)
           this->participant_factory_ = TheParticipantFactory;
           TheServiceParticipant->set_default_discovery (OpenDDS::DCPS::Discovery::DEFAULT_RTPS);
 
+          using namespace OpenDDS::DCPS;
```

Comment:
I prefer not to use `using namespace`, could you remove this and just use the full scoping as we did originally?